README.md

```diff
@@ -1,16 +1,6 @@
 # StakaterPlatform
 
-## Overview
-
-This section provides a bird eye view of StakaterPlatform.
-
-### Problem Statement
-
-How can a novice use can deploy tools that is required for the management of kubernetes cluster?
-
-### Solution
-
-StakaterPlatform provides out of the box stacks to control, monitor, log, trace and security for applications deployed on kubernetes using CI/CD pipelines.
+StakaterPlatform Supercharges kubernetes clusters to provides out of the box stacks to control, monitor, log, trace and security tools.
```

Comment:
i think here we have some grammatical mistake

Reply:
Some issue in PR it's not showing correct changes in this file.